app.js

```diff
@@ -214,6 +214,16 @@
             options.password = argv.password;
         }
         
+        if (argv.photoshopVersion && typeof argv.photoshopVersion === "string") {
+            options.photoshopVersion = argv.photoshopVersion;
+        }
+        if (argv.photoshopPath && typeof argv.photoshopPath === "string") {
+            options.photoshopPath = argv.photoshopPath;
+        }
+        if (argv.photoshopBinaryPath && typeof argv.photoshopBinaryPath === "string") {
+            options.photoshopBinaryPath = argv.photoshopBinaryPath;
+        }
+    
```

Comment:
These arguments should be documented in the optionParser's `describe` call around line 62. They don't need aliases, though.